### DISCUSSION TITLE
Error:import dbeaver to eclipse !

### DISCUSSION THREAD

**Discussion by xikeda-hunzi** [dbeaver/dbeaver#16941] [DBeaver Development]
I had some problems following this procedure:
https://github.com/dbeaver/dbeaver/wiki/Develop-in-Eclipse
when i import the maven project, some packages cannot be downloaded:
<img width="271" alt="886141a13c53a974260e8836850a124" src="https://user-images.githubusercontent.com/91422515/176846306-5dbd1b3e-e441-4210-8447-7afc6ea2ccaa.png">
<img width="399" alt="cdf201a8ddf0b1fe121263ba93beb8b" src="https://user-images.githubusercontent.com/91422515/176846331-a431bed4-084f-413d-b706-c5485ef3e819.png">

**Comment by miltonchirinos44** [comment]
make sure you use java 11 and maven 3.6.3 and configure eclipse that way since by default it is with the latest version of java or maven

java
![image](https://user-images.githubusercontent.com/52106718/184423766-4f3c28aa-68f0-48c7-8ed0-ba5b3855402e.png)

maven
![image](https://user-images.githubusercontent.com/52106718/184423892-67596ec6-4b3f-482d-8f42-02df6004f0f7.png)
